walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,67 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  font-family: Arial, sans-serif;
+  padding: 4px;
+  text-align: center;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+h1 {
+  text-align: center;
+  font-weight: normal;
+}
+
+nav {
+  background: white;
+  width: 100%;
+  margin-bottom: 20px;
+  position: fixed;
+  top: 0;
+  padding-top: 1vw;
```

Comment:
Be careful with using `vw`. It can really backfire if for example you had an ultra-wide monitor in which case this top padding would be way too big. 

It's more preferable to have media breakpoints like you've defined below for different device sizes. You could then easily define this padding should be 16px on desktop, 8px on tablet, and 4px on mobile or something like that.